sdk/cosmos/azure-cosmos-spark_3-5_2-12/src/test/scala/com/azure/cosmos/spark/CosmosCatalogITest.scala

```diff
@@ -70,6 +70,23 @@ class CosmosCatalogITest
         }
     }
 
+    it can "list all databases" in {
+        val databaseName1 = getAutoCleanableDatabaseName
+        val databaseName2 = getAutoCleanableDatabaseName
+
+        // creating those databases ahead of time
+        cosmosClient.createDatabase(databaseName1).block()
+        cosmosClient.createDatabase(databaseName2).block()
+
+        val databases  = spark.sql("SHOW DATABASES IN testCatalog").collect()
+        databases.size should be >= 2
```

Comment:
== 2?

Reply:
it is possible there are other databases has been created by other tests, so using >= 2 + verify those two new created ones are returned.